paymentsheet/src/main/java/com/stripe/android/link/attestation/DefaultLinkAttestationCheck.kt

```diff
@@ -35,7 +35,8 @@ internal class DefaultLinkAttestationCheck @Inject constructor(
                     val lookupResult = linkAuth.lookUp(
                         email = email,
                         emailSource = EmailSource.CUSTOMER_OBJECT,
-                        startSession = false
+                        startSession = false,
+                        customerId = linkConfiguration.customerId
```

Comment:
attestation doesn't need `customerId`, right?

Reply:
it does - in prod we'll use the mobile/lookup endpoint for verified merchants as an extra layer of security too

Reply:
Correct if I'm wrong, but in this request in particular, it's only used for attestation. `linkAuth.lookUp()` obtains and passes an integrity check token, makes a request, and largely drops the result. Unless the customer ID is used in the backend for attestation, I don't think it's necessary.